app/src/Components/Error/GraphQLError.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { UnauthorisedError } from './UnauthorisedError';
+
+class GraphQLError extends React.Component {
+  render() {
+    const error = this.props.error;
+    if (error.graphQLErrors.length > 0) {
+      return <div>
+        {error.graphQLErrors.statusCode + ': ' + error.graphQLErrors.result.message}
```

Comment:
Is the variable graphQLErrors a single error or an array?

Reply:
graphql errors is an array, because your querry can have multiple issues, i guess. 

It is different from network error which is a single object.